src/cli/field_data.rs

```diff
@@ -0,0 +1,30 @@
+use anyhow::{bail, Result};
+use serde::{Deserialize, Serialize};
+
+use lurk::field::{LanguageField, LurkField};
+
+/// A wrapper for data whose deserialization depends on a certain LurkField
+#[derive(Serialize, Deserialize)]
+pub struct FieldData {
+    pub(crate) field: LanguageField,
+    data: Vec<u8>,
```

Comment:
Please annotate with `serde_bytes` or some equivalent method (preferably `#[serde_with::serde_as(as = "Bytes")]` which we probably use elsewhere), for [performance](https://docs.rs/serde_bytes/latest/serde_bytes/)